homeassistant/helpers/restore_state.py

```diff
@@ -63,7 +65,7 @@ def async_get_last_state(hass, entity_id: str):
             yield from hass.loop.run_in_executor(
                 None, _load_restore_cache, hass)
 
-    return hass.data[DATA_RESTORE_CACHE].get(entity_id)
+    return hass.data.get(DATA_RESTORE_CACHE, {}).get(entity_id)
```

Comment:
Why did you change this? There are no paths in `_load_restore_cache` that do not set the cache